backend/test/deal-observer.test.js

```diff
@@ -74,3 +76,40 @@ describe('deal-observer-backend', () => {
     assert.deepStrictEqual(expected, actual)
   })
 })
+
+describe('deal-observer-backend binary', () => {
+  let pgPool
+  before(async () => {
+    pgPool = await createPgPool()
+    await migrateWithPgClient(pgPool)
+  })
+  beforeEach(async () => {
+    await pgPool.query('DELETE FROM active_deals')
+  })
+  after(async () => {
+    await pgPool.query('DELETE FROM active_deals')
+  })
```

Comment:
Would you mind not cleaning the DB after tests in this pull request and waiting until we resolve the discussion in https://github.com/filecoin-station/deal-observer/pull/41?